src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1287,6 +1287,22 @@ The following important rules apply to global variables and generic information:
             </genericInformation>
 ----
 
+=== Manage job labels
+
+A label is a property that can be attached to a job.
+The advantage of having labels is given by the possibility to group and manipulate a certain number of jobs by filtering based on this "custom status" or label.
+
+The job labels are stored in a table in the scheduler database.
+When a label is attached to a job, the label information will be stored in the job table in the scheduler database.
+
+It is possible to define the job labels list in the Admin menu from Scheduler Portal in order to be able to set a label on a job.
```

Comment:
Replace the two paragraphs above with:

Administrators can define which labels can be applied to jobs in the *Admin* menu of the Scheduler Portal (link to glossary). Physically, possible labels are stored in the scheduler database.